src/middleware/rateLimiter.ts

```
@@ -138,78 +138,125 @@ class MongoRateLimitStore implements Store {
+    try {
+      const db = getMongoDB();
+      const collection = db.collection<RateLimitDocument>(RATE_LIMIT_COLLECTION);
+      await collection.updateOne(
+        { key: this.buildKey(key) },
+        {
+          $inc: { "value.count": -1 },
+          $set: { updatedAt: new Date() },
+        },
+      );
+    } catch (error) {
+      logger.warn("MongoRateLimitStore.decrement failed, skipping (in-memory fallback has no decrement)", {
```

ESLint

[error] Replace `"MongoRateLimitStore.decrement·failed,·skipping·(in-memory·fallback·has·no·decrement)",` with `⏎········"MongoRateLimitStore.decrement·failed,·skipping·(in-memory·fallback·has·no·decrement)",⏎·······`

(prettier/prettier)